SingleGUI.java

```diff
@@ -109,6 +142,30 @@ private void backButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
         // TODO add your handling code here:
     }//GEN-LAST:event_backButtonActionPerformed
 
+    private void fileChooserActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_fileChooserActionPerformed
+        // TODO add your handling code here:
+    }//GEN-LAST:event_fileChooserActionPerformed
+
+    private void cmdLnArgActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_cmdLnArgActionPerformed
+        // TODO add your handling code here:
+    }//GEN-LAST:event_cmdLnArgActionPerformed
+
+    private void exptdOutputActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_exptdOutputActionPerformed
+        
+        
+
+// TODO add your handling code here:
+    }//GEN-LAST:event_exptdOutputActionPerformed
+
+    private void runCompileActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_runCompileActionPerformed
+
+        runCompile.getText();
```

Comment:
you should be getting the values of the two textfields you added and storing them in variables here